<launch>

  <node name="calibrate_kinect_checkerboard" pkg="clam_vision" type="calibrate_kinect_checkerboard" output="screen">
    <param name="camera_frame" value="/camera_link" />
    <param name="fixed_frame" value="/base_link" />
    <param name="tip_frame" value="/gripper_fake_tip_link" />
    <param name="gripper_tip_x" value="0.0" />
    <param name="gripper_tip_y" value="0.0" />
    <param name="gripper_tip_z" value="0.0" />
  </node>

  <node name="checkerboard_image_view" pkg="image_view" type="image_view">
    <remap from="image" to="/calibrate_kinect_checkerboard/calibration_pattern_out" />
    <param name="autosize" value="true" />
  </node>
</launch>
